common/smoke-test/smoke-tests.yml

```diff
@@ -26,12 +26,10 @@ jobs:
           OSVmImage: "ubuntu-18.04"
           CloudType: AzureChinaCloud
           ArmTemplateParameters: $(azureChinaCloudArmParameters)
-          Location: "chinanorth"
```

Comment:
Why is the location parameter deleted?

Reply:
This was overriding the default location. If no location is specified the default location is selected based on the cloud: https://github.com/Azure/azure-sdk-tools/blob/master/eng/common/TestResources/New-TestResources.ps1#L132